inc/customizer/class-spacious-customizer-partials.php

```diff
@@ -0,0 +1,111 @@
+<?php
+/**
+ * Spacious customizer class for theme customize partials.
+ *
+ * Class Spacious_Customizer_Partials
+ *
+ * @package    ThemeGrill
+ * @subpackage Spacious
+ * @since      Spacious 2.4.5
+ */
+
+// Exit if accessed directly.
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+/**
+ * Spacious customizer class for theme customize partials.
+ *
+ * Class Spacious_Customizer_Partials
+ */
+class Spacious_Customizer_Partials {
+
+	/**
+	 * Render the Read Next text for selective refresh partial.
+	 *
+	 * @return void
+	 */
+	public static function read_more_text_render() { ?>
+		Hello!
+		<a class="read-more" href="<?php the_permalink(); ?>"><?php echo esc_html( get_theme_mod( 'spacious_read_more_text', __( 'Read more', 'spacious' ) ) ); ?></a>
+	<?php }
+
+	public static function spacious_header_info_text() {
+		if ( get_theme_mod( 'spacious_header_info_text', '' ) == '' ) {
+			return;
+		}
+
+		$spacious_header_info_text = '<div class="small-info-text"><p>' . get_theme_mod( 'spacious_header_info_text', '' ) . '</p></div>';
+
+		echo do_shortcode( $spacious_header_info_text );
+	}
+
+	public static function spacious_breadcrumb() {
+
+		// NavXT
+		if ( function_exists( 'bcn_display' ) ) {
+
+			echo '<div class="breadcrumb" xmlns:v="http://rdf.data-vocabulary.org/#">';
+			echo '<span class="breadcrumb-title">' . wp_kses_post( get_theme_mod( 'spacious_custom_breadcrumb_text', __( 'You are here: ', 'spacious' ) ) ) . '</span>';
+			bcn_display();
+			echo '</div> <!-- .breadcrumb : NavXT -->';
+
+		} else if ( function_exists( 'yoast_breadcrumb' ) ) { // SEO by Yoast
```

Comment:
also https://developer.wordpress.org/coding-standards/wordpress-coding-standards/php/#use-elseif-not-else-if